fibonacci.js

```diff
@@ -0,0 +1,162 @@
+const loader = document.getElementById("loader");
+const loader2 = document.getElementById("loader2");
+let btn = document.getElementById("btn");
+let result = document.getElementById("result");
+let checked = document.getElementById("flexCheckDefault");
+let dropdown = document.querySelectorAll(".dropdown-item");
+let history = document.getElementById("history");
+
+function removeSpinner() {
+  loader.classList.remove("spinner-border");
+}
+function removeSpinner2() {
+  loader2.classList.remove("spinner-border");
+}
+
+function callServer(number) {
+  fetch(`http://localhost:5050/fibonacci/${number}`)
+    .then((response) => {
+      if (!response.ok) {
+        response.text().then((erText) => {
+          result.innerHTML = "Server Error:" + " " + erText;
```

Comment:
You should throw an error with the erText and have a .catch. You should display the error in the .catch: 

`throw new Error(erText)`

`.catch(err => {
result.innerHTML = "Server Error:" + " " + erText;
 result.classList.add("text-danger");
 removeSpinner();
 removeSpinner2();
})`